src/GeositeFramework/css/main.css

```diff
@@ -2193,6 +2202,16 @@ body.x-body {
         width: 100vw !important;
     }
 
+    .arcgisSearch.searchExpanded .searchBtn {
+        margin-right: 2px;
+    }
+
+    .arcgisSearch .searchBtn {
+        padding: 8px 4px 6px 4px;
+        height: 25px;
+        width: 34px;
```

Comment:
Do we have any flexibility to increase this to 40x40 px?

Reply:
I increased this a bit more within the allotted space. The target is now 46px x 39px.